.github/workflows/aws_tfhe_backward_compat_tests.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     name: Backward compatibility tests
     needs: [ setup-instance ]
     concurrency:
-      group: ${{ github.workflow }}_${{ github.head_ref || github.ref }}
+      group: ${{ github.workflow_ref }}
```

Comment:
what is workflow ref ? vs github.ref we were using before the PR target thing ?

Reply:
Before we were using two context variable:

* `github.workflow` uses what's in `name:` directive at the top of the workflow. If there is none it falls back to workflow path.
* `github.ref` will print the branch in the form `refs/heads/my_dev_branch`

Now we can do the same with only one:

* `github.workflow_ref` will display `path/to/workflow@refs/heads/my_dev_branch`

Reply:
neat!